package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "bytes",
   "description": "Utility to parse a string bytes to bytes and vice-versa",
-  "version": "2.4.0",
+  "version": "3.0.0",
```

Comment:
Typically version number changes are not great in a PR, as I can never merge the PR with it in there, since I cannot do the release prep or merge in other PRs afterwareds.